i18n/bg.json

```diff
@@ -2473,7 +2473,7 @@
   },
   {
     "id": "app.system.warn_metric.bot_description",
-    "translation": "[Научете повече за Mattermost съветника](https://about.mattermost.com/default-channel-handle-documentation)"
+    "translation": "[Научете повече за Mattermost съветника](https://docs.mattermost.com/manage/in-product-notices.html#admin-advisor-notices)"
```

Comment:
(non-blocking for me) I believe we are only changing translation strings at `en.json` and Translation team will follow through the necessary changes per language owner. So, I don't know if this could be an exception.